src/main/java/uk/ac/ic/wlgitbridge/snapshot/base/MissingRepositoryException.java

```diff
@@ -15,11 +15,30 @@
         "If this problem persists, please contact us."
     );
 
-    public static final List<String> EXPORTED_TO_V2 = Arrays.asList(
-        "This Overleaf project has been moved to Overleaf v2, and git access is temporarily unsupported.",
-        "",
-        "See https://www.overleaf.com/help/342 for more information."
-    );
+    static List<String> buildExportedToV2Message(String remoteUrl) {
+        if (remoteUrl == null) {
+            return Arrays.asList(
+                    "This Overleaf project has been moved to Overleaf v2, and cannot be used with git at this time.",
+                    "",
+                    "If this error persists, please contact us at support@overleaf.com, or",
+                    "see https://www.overleaf.com/help/342 for more information."
+            );
+        } else {
+            return Arrays.asList(
+                    "This Overleaf project has been moved to Overleaf v2, You should change your remote to:",
```

Comment:
Could we update this line(s) to something that includes a brief explanation of why it needs to be updated:

```
This project has been moved to Overleaf v2 and has a new identifier.
Please update your remote to:
```

Reply:
Certainly :+1: